PositiveAirwayPressureDevices/R4/files/PositiveAirwayPressureDevicesPrepopulation-0.1.0.cql

```diff
@@ -47,6 +55,33 @@ define PapDeviceRequested:
   else if  "DeviceRequestHcpcsCoding".code.value = 'E0601' then 'E0601'
   else 'null'
 
+define "Today": Today()
+
+define "True": true
+
+define "False": false
+
+define "AHI": HighestObservation(WithUnit(Verified(ObservationLookBack([Observation: "AHI_Codes"], 3 months)), '/h'))
+define "RDI": HighestObservation(WithUnit(Verified(ObservationLookBack([Observation: "RDI_Codes"], 3 months)), '/h'))
+
+define "AllAlergy": ActiveAllergy([AllergyIntolerance])
+
+define "AllAllergyWithCode": 
+  distinct(flatten(
+    [AllergyIntolerance] A
+      let Codings: 
+        (A.code.coding) CODING where CODING.system.value in {
+          'http://snomed.info/sct'
```

Comment:
Allergy should use RxNorm codes

Reply:
There are multiple types of allergy. We probably should include all of them. RxNorm is for drug allergy. 

https://www.hl7.org/fhir/v2/0127/index.html

Here are more granulated codes for animal allergies:
https://phinvads.cdc.gov/vads/ViewCodeSystemConcept.action?oid=2.16.840.1.113883.6.96&code=717234006

Probably need to use a value set.

Reply:
In CDE: ALL: Allergies (include RxNorm if Known). My understanding is that only include medication allergies. Include RxNorm code if known.